css/_premeeting-screens.scss

```diff
@@ -1,22 +1,30 @@
 /**
  * Shared style for full screen local track based dialogs/modals.
  */
+ .premeeting-screen,
```

Comment:
@vp8x8 Can you please review the CSS here, and the changes to the Prejoin component? The latter LGTM, but an extra pair of eyes won't hurt.

Reply:
LGTM